### DISCUSSION TITLE
Why Proof of Stake is the antithesis of cryptocurrency

### DISCUSSION THREAD

**Discussion by GiverofMemory** [dogecoin/dogecoin#2835] [General]
**Current proposal: Hybrid ASIC/CPU mining.  Start with 90% scrypt and 10% randomx/yespower and over the years slowly equalizing the percentages.**

More discussion: https://np.reddit.com/r/dogecoindev/comments/sjb5vn/doge_improvement_proposal_001_hybrid_pow/

I have unfortunate news for all of us.  The government(s) are coming for crypto.  Those of us who have been in this for a while are being shocked day by day the intrusions into this industry that governments are making, and things are getting very close to home.

The biggest attack on this industry is PoS.  I know many have good intentions for this with the environment and things like that.  However it is not a free and open (or secure) protocol.  Cryptocurrencies aren't created on an exchange.  Every single bitcoin has been mined anonymously.  Only later do these bitcoins end up on exchanges.  The exchange is the regulatable part by governments.  What proof of stake does is basically place coin creation on the exchange.  How do you enter a proof of stake protocol?  You have to buy coins.  This is exactly what the governments would like, you have to buy coins to make coins and the made coins are connected to your purchased coins.  In proof of work no exchanges are needed.  This is the true future of a breakaway financial system - every merchant accepts crypto so you can earn it or mine it and spend it, not need to buy it with fiat.  Proof of stake is just legacy banking 2.0 and will be no different than the current system with all the KYC and regulation that cryptocurrency was created to obsolete.  Also once the large cryptos go proof of stake then the SEC will likely be able to regulate them directly as securities, this is the trojan horse.  They won't tell you that's the plan until most coins already switch to PoS and get trapped.

So if you like fiat and banks then you want PoS.  If you want the antidote to that you want PoW.  Now there are ways to make PoW more efficient and the essence is CPU only mining, which was honestly what dogecoin was designed to do.  One way to do this is with relatively advanced algorithms like Yespower (upgraded scrypt) or RandomX.  Another way is by increasing the size of the dataset to be hashed over time to resist asics and GPU's that way.  Anyway scrypt was the most asic resistant (energy efficient) algo at the time and it would only make sense to go that direction for dogecoins energy efficient future.

**Comment by rnicoll** [comment]
Your... argument is that it's better that people with the resources to mass-mine Dogecoin make them, rather than the people with resources to stake can get Dogecoin AND people on exchanges can stake their coins? I mean I actually do see how there being an origin point for each Dogecoin (and the exchange likely reporting where it goes next) does change the dynamic, but isn't this more an argument for not keeping funds on exchanges?

**Reply by dxteam2024** [reply]
People prefer to put doge on exchanges because it has not achieved the effect of universal use. Pos does not make people more willing to pledge, people are actually lazy, and it is relatively convenient to put on exchanges. Only when doge can be used at will, people will not put it on exchanges or POS.

**Reply by dxteam2024** [reply]
It's easier for someone with resources to get a huge amount of DOGE, but it's relatively more expensive to get a huge amount of electricity and machines。POS makes it easier to be a ”Controller“，due to the existence of the ”controller“, the efficiency will actually be lower .

**Reply by dxteam2024** [reply]
POS is mathematically more efficient, but for the financial system, there are more social things to consider. POS actually complicates the entire financial system so much that its efficiency is actually no better for the waste of its complexity

**Comment by tealethetoolman** [comment]
I agree with OP but for different reason. 
i see that op and I both love randomx and the implementation it has in monero allowing me to mine on my cpu dense machines to heat my house. CPUs are easy to get and so this is great, but definitely not the answer because it would destroy the pow hash power. It’s too late for that.

i agree that PoS is antithetical to crypto as well. Right now the bar for manipulation of data (double spend or transaction muting)  is pretty high. Right now you would need to own a fab creating revolutionary hardware, and a way to power enough of it to more than double the size of the network. With proof of stake, you just need a bad actor to spin up enough nodes, a collaboration between janet yellen and jeff bezos of ious for liquid cash to buy coins to stake and point everyone at bad nodes where transactions are muted or double spent. Thats a very low bar. We already see how easy it is for michael saylor to buy up large swaths of bitcoin for in hopes that he’ll be king of the new world, so it wouldn’t be hard to use the same strategy to defeat the network itself. 

Then there’s two other important facts that we need to discuss that proof of work allows for that we need. 

With automation and raising minimum wages, we’re changing two different bars of entry to the workforce, plus combined with all if the disabled and burnt out people, there are a lot of people who will be excluded from the economy. We need a ubi. PoW takes electricity from the largest machine humans have invented and turning it into secure money in the same fashion as a lottery. The more hash power you have, the better your chances of winning that lottery. The fixed inflation is great because it makes sure there will be continuous liquidity.  This would further make a good case for randomx as it allows more people to mine with existing hardware and not having to buy hardware, but as I mentioned before that would have a detrimental effect on the network security. 
Second lets address the ESG folks. I know that PoW uses a lot of energy and the eco fascists are pressuring us to shut it down not realizing this is a global network. The energy isn’t a bad effect of PoW but actually a side affect of a horrible global patent law that stops nations from being able to make their own hardware for dependable clean electricity. GE, Siemens own all of the patents for production, transmission, and distribution, and lockheed matin owns all the patents for near future energy generation technology developed in the 60s-80s using tax money for space exploration purposes- all shelved and guarded by an army of lawyers. What makes it worse is they find every method through extensive and well (tax) funded r&d with a complimentary army of lawyers to enforce these patents. In the USA we have FERC and RTOs messing with the stability of our grid with all of their conflicting rules, the eco fascists pushing for wind and solar while neglecting that every mwh of solar/wind needs to be backed by a complimentary mwh of fossil energy thus driving our energy costs up and stability down. This is something beyond us to fix (maybe time for someone to make a dao to protect people so they can use these shelved patents freely). 

Now comments on proof of stake (and the person pushing for it the most)
Vitalik. 
who put him in the foundation? Why is he doing back door meetings to try to steal doge from under us? 
personally i believe he needs to go fix the problems with ethereum and the insane cost of network operations before we let him anywhere near our coins. He’s a huge proponent of PoS and has been working on it for years to no avail. We’re not his testbed. Nobody should have to pay insane fees to an unbalanced democracy of his creating (unbalanced because in his world, miners have 100% vote and users 0% vote which favors code versions that make miners more money like in fees). While I agree that PoS is good for speculators who want to grow their money doing nothing but staking, it’s not good for a network of people trying to make a global money. (eth might as well be related to an aws giftcard with massive usage fees). 

On the other hand, Michi made a good point that with our current PoW implementation, mining is centralized around pools that control the code checkouts of their constituents, and the warehouses full of asics. This is a problem, but at least they are securing the network. It makes it unfair to the rest of us who want to mine it and be a part of the UBI lottery and I get this frustration. How hard is it to get your hands on these asics so that we can independently contribute to the network? It’s pretty easy to get your hands on a raspberry pi or a dell poweredge r910. In the end, the people who run these massive warehouses with their wealth accumulations will be the saylors of doge/ltc (since they're auxpow’d together). This is a legitimate problem. randomx can solve this problem but by sacrificing then strength of the network as everyone ramps up to get involved. 

In conclusion, these are a series of components that we as a community need to come together to hash out. We need to respect the balance between network strength, stopping the no-coiners from being able to leverage the dollar printer to pollute the data, and most importantly keeping the fees for using doge cheap. Vitalik is toxic to our community- especially with these back room ideas he’s been up to. I personally feel very strong about this and after my talk at defcon last year, I know that I’m not alone with these thoughts. 

Final thought: if people want a PoS dog themed coin, go use shiba when ethereum turns proof of stake. See if it’s usable (even now) off of exchanges.

**Reply by patricklodder** [reply]
> PoW takes electricity from the largest machine humans have invented and turning it into secure money in the same fashion as a lottery.

During bull markets, yes. During bear markets, not so much. Over time, equilibrium is found and mining becomes a zero sum game. Profits are made during the periods of imbalance - it's not a reliable source of income because it depends on the volatility of secondary markets. Therefore it's less interesting to compare securing a blockchain to UBI, because it only works if DOGE would always, guaranteed, trade at a premium versus equipment depreciation and operating costs. I think that in 2014 this was already proven to not be the case.

> mining is centralized around pools that control the code checkouts of their constituents, and the warehouses full of asics

The fact that PoW can get industrialized can also be asserted for PoS. Look at the [top validators for Cosmos Hub](https://atomscan.com/validators) or even [ultimate operators of top Cardano staking pools](https://cardanoscan.io/pools?inActive=true) and you'll see this pattern too. It's all very top-heavy with industrial players and now-rich early adopters. It's not designed for those late to the party unless you bring an above-average amount of cash with you. Neither is DOGE fwiw, it was "designed" to be dead by block 600k, just not coded to do that "by accident".

> most importantly keeping the fees for using doge cheap.

In proof-of-stake with low usage fees, any inflation is both paying stakers and cancelling out (part of) their profit. In the utopian scenario where every DOGE is secured by 1 DOGE (and no one utilizes it), the inflationary element equals the income. Anyone that doesn't stake is hurting their future bottom line in favor of the bottom line of those that do, unless they find more profitable means to activate their assets. 

Assume 51% honestly and consistently staked and no evil players stake, which is the absolute minimum on the current most secure PoS chain: if inflation is 4%, APY is just under 8%, but net profit will be only 4% due to the same inflation, less if a higher % stakes, eg "evil banks" that need to be offset by "honest shibes" will require a higher % of coin staked. That means that any alternative capable of 12% APY creates double the returns in the best case scenario currently known for staking.

Further assuming that block reward stays subsidy + fees, the only way to make it more competitive to secure the chain becomes to either increase inflation, or increase fees; I would have expected more dissenting comments on the fee proposal if that is the vision. Increasing inflation further is probably not favorable as long as DOGE is not the universe's reserve currency.

----

Since this entire PoS/PoW discussion is political and based on opinions rather than verifiable information, let's take a step back. We're having a discussion with a bunch of shibes [here](https://np.reddit.com/r/dogecoindev/comments/sjb5vn/doge_improvement_proposal_001_hybrid_pow/hvu4t2q/?context=3) which is now going into a direction of actually doing some research before proposing that "PoS is the way". I have yet to see proof that we don't have time to do this right, so let's at the very least try to get some quantifiable data before we start hyping solutions.

I'm considering @mishaboar's suggestions there and right now do feel there is a good case to create a proposal to fund some research, here.

_edit: fixed a badly written comparison, oops_

**Reply by IDK1964** [reply]
POS will do nothing but put the control in the hands of big money Exchanges. POS is and will never be good for DOGE. Pat I thought you out off all were the smart one.

How quick to forget the community that got DOGE to where it's at.

Not to mention make mining taxable now in the form of capital gains.

**Reply by patricklodder** [reply]
> How quick to forget the community that got DOGE to where it's at.

Let me TLDR my message for you because I guess I didn't make myself clear: Threat of centralization is probably worse for PoS than for PoW. There is no real problem identified except that it is currently unknown how much securing Dogecoin pollutes. As long as there is no data and problem statement that can be validated, it makes no sense to offer solutions that require forking current consensus mechanisms. So all I propose is to start collecting some data, so that problems can be assessed. 

Could you please explain to me in which way that statement upsets you?

**Reply by IDK1964** [reply]
Sorry my bad I must have missed that part... 
But here is something that adds to my comment with the exception of the misunderstanding: https://www.reddit.com/r/dogecoin/comments/smuhbb/pow_vs_pos_doge/?utm_source=share&utm_medium=web2x&context=3

**Reply by IDK1964** [reply]
If you're going to do anything with DOGE it should be to follow it's path of making fun of BTC/crypto and instead of halving the block reward and increasing the diff, DOGE should halve its block times and lower its diff. Let the Network and internet latency dictate the blocks. The market will set its value not the coins design.

**Reply by GiverofMemory** [reply]
> This would further make a good case for randomx as it allows more people to mine with existing hardware and not having to buy hardware, but as I mentioned before that would have a detrimental effect on the network security

To make the switch from Scrypt to any algo would be "detrimental to security" in the short term so ya there would need to be multiple year lead times before a switch to allow people to gear up in new equipment to make the switch.  However, hashpower follows adoption and market value so say we were to give 3 years notice that we were changing to something like RandomX then when it actually went into effect it would be quite secure.

**Reply by GiverofMemory** [reply]
> If your going to do anything with DOGE it should be to follow it's path of making fun of BTC/crypto and instead of halving the block reward and increasing the diff, DOGE should halve its block times and lower its diff. Let the Network and internet latency dictate the blocks. The market will set its value not the coins design.

Yes fun, quick blocks, potentially larger blocks, and potentially more asic resistant (which was the original purpose of scrypt) should be the path forward imo. 

What hybrid I would be much more supportive of is hybrid between scrypt and randomx/yespower.  Starting off with 90% scrypt 10% cpu algo and slowly equalizing the ratio over time.

**Reply by IDK1964** [reply]
And another thing, AuxPoW needs to go.

**Reply by patricklodder** [reply]
> AuxPoW needs to go.

To solve what problem?

**Reply by IDK1964** [reply]
To solve what problem?
The anchor LTC has become.

**Reply by IDK1964** [reply]
Patrick. Suggestion; create a DOGE only pool and promote it and the pool fee could cover the Dev tip jar.

**Reply by patricklodder** [reply]
> create a DOGE only pool and promote it and the pool fee could cover the Dev tip jar.

Gave this some thought; bear with me please.

I think you should change the idea from DOGE-only to DOGE-centric, because I think the former is masochistic and wasteful - I don't see there being a point to throwing away potential income by not using AuxPoW, and tribalizing hash power sounds like a step back from what we've got right now. I read your problem with AuxPoW to be political rather than technical: the current focus of pools is mainly on Litecoin, and less so on Dogecoin. All this means is that, now that Dogecoin is contributing a significantly larger share of the total scrypt rewards, there is an opportunity to launch and market a pool that centers around Dogecoin, aligns closer to the Dogecoin block creation timing rather than the Litecoin (or DGB-scrypt) timing in terms of resets (see also the discussion some shibes had with Litecoinpool about empty blocks and work resets.)

As for me personally running that, long-term that's a hard no. The amount of time I am spending on Dogecoin Core has been ridiculous already and it's not going to get less soon unless I also abandon you - which I don't want to do. What I can do is set up an (advanced) p2pool-kind-of-network. I think that road would have the most merit because it's decentralized. The size of a donation (if any) to a tipjar would then be up to the miners, where it belongs. To start this off, I'll spend some time on reviewing proposals like [the braidpool paper](https://github.com/pool2win/braidpool/raw/main/proposal/proposal.pdf) and see what can be done to have a more modern, inclusive p2p mining solution.

**Reply by IDK1964** [reply]
I personally have no problems with LTC. It's just from my personal observations it seems to me LTC has become somewhat of an anchor, higher diff slower block times. Look at litecoinpool.org they don't contribute to the DOGE network much at all if any...more like they use DOGE.?. Please correct me if I'm wrong.

2nd matter: as far as DOGE only I was actually thinking more DOGE "Official" with ur name on it. And you do you, add all the coinz you want. Just don't turn it into a slightly different version of Prohash. Look to hyperdonkey.com for inspiration.js

**Reply by IDK1964** [reply]
Yeah, Priorities are priorities.
Maybe you could get yourself an apprentice.l0l/just HODL ur DOGE.?.
 If I could do it, I would((Help)). If you could just get it up I'll handle some of the promotion.?.

**Reply by patricklodder** [reply]
> I was actually thinking more DOGE "Official" with ur name on it

From where I'm sitting, there is no such thing as official; there is only unofficial. This is because Dogecoin is not some stupid token, it's an actual permissionless cryptocurrency and I know no one, including myself, that could be trusted with an official function anyway - and it's also not needed. I consider any claims otherwise as frivolous and self-serving for the claiming party; it's just people looking trying to gain power and/or money and/or fame, for them. Otherwise they would come up with a solution where they wouldn't need all that power. Also, according to human history, even the most skillful dictators (or their offspring) will see their power erode over time - and no one nearly as excellent as a real world emperor has ever tried to rule Dogecoin, it's only the noobs and scammers that try.

Instead, if we make this an open source p2p thing, there is no need to trust anyone. I think that would do Dogecoin way more justice than yet another claim to fame that will just erode and give people the opportunity to eff up. If that means that all the usual suspects will stay far away from it and will not lend their support, then that's a blessing and it means we're doing something right. May be harder than packaging straight lies in a wrapping of doing only good, but that doesn't make it worse.

I'll give the entire thing some more thought and come back with something concrete to chew on... like working code 😁

**Reply by IDK1964** [reply]
lol. Works for me.

**Comment by IDK1964** [comment]
If DOGE goes POS, you might as well kiss it goodbye.

**Comment by Techsimp68** [comment]
Reading this is killing me XD PoW or PoS doesn’t have anything to do with “going into the back door to steal doge” that’s doge’s lack of security. It is the worse crypto in that aspect, if doge changes to another concept, it’ll fall back to where it was in 2013. Also changing the concept will not affect the government "coming after doge" (or crypto) they will find a way to monitor any action, 90% block chains are not anonymous anymore, the government changed that years ago it's difficult to follow but isn't anonymous. So, thinking changing the concept to hide from the government shows the lack of seeing the big picture or knowing the rules. So, it can hit close to home all it wants not much you can do to hide your transactions if they are on the exchange or not.

Doge isn't made for PoS people will drop it if it changes just a data analytics/Block chain specialist two cents. To add I'm not a fan of doge so if it dies again or not doesn't affect me at all

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by patricklodder** [reply]
If privacy/fungibility is an issue you'd want to address, I think we should have a discussion about that on its own - it's basically a question of what the consequences are of bringing in MWEB, ring signatures, dash-style coinjoin or zero knowledge spends. I feel it's hard to predict what the long-term effects will be, even (or maybe especially) after talking about MWEB with Charlie last year, because how do we de-taint coin if the mixing itself is the cause of the taint... a lot of the compliant US crypto services have banned any utxo that have a mixer in their history. Privacy by default feels super-risky to introduce on a whim because there's no way back, and optional privacy would mean that you risk people opting in to getting fungible, but unspendable coins.

Maybe DashPay devs know how to do it since they're supposedly working with Visa and I'd assume they have looked at US AML law compliance?

**Reply by GiverofMemory** [reply]
Satoshi was a genius.  Bitcoin is not 100% anonymous as soon as exchanges are in the picture, but it is entirely possible to use bitcoin or dogecoin very privately right now, use xpub's to collect payments (every single transaction is unlinkable) and miners mining coins anonymously, it is good enough honestly if you use good hygiene and stay away from exchanges.

**Comment by Techsimp68** [comment]
Doge isn’t Anonymous.
A lot of forms and information can tell you this.

https://dogecoinisawesome.com/faq#is-dogecoin-anonymous

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Reply by samsawyerz** [reply]
I am forever grateful to RECOVERY EXPERTS [RECOVERY EXPERTS @ CYBERSERVICES dot com] for helping me recover my STOLEN BTC. I will continue to recommend them to anyone in need of FUNDS RECOVERY.

**Comment by Techsimp68** [comment]
xpub's Doesn’t hide transactions it forwards them to other wallets which in turn can be traced.

"The difficulty with UTXOs above is that in order to track your balances with regular public keys, you need to add every single wallet address from every transaction"

"Anonymity- describes situations where the acting person's identity is unknown." So if the transaction can be traced with key's it's not "Anonymous"

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
